wagtail/tests/test_sites.py

```diff
@@ -172,18 +173,14 @@ def test_there_can_only_be_one(self):
             site.clean_fields()
 
     def test_oops_there_is_more_than_one(self):
+        # It will through an error: django.db.utils.IntegrityError when multiple sites with is_default_site created
```

Comment:
```suggestion
        # It will throw an error: django.db.utils.IntegrityError when multiple sites with is_default_site created
```

I felt "through" is a typo. Check once, please.

Reply:
fixed, thanks!

Reply:
Happy to help :)